frontend/public/locales/es/translation.json

```diff
@@ -0,0 +1,279 @@
+  "Back": "Back",
+  "Canary: {{isCanary}}": "Canary: {{isCanary}}",
+  "Cancel": "Cancel",
+  "Choose the traffic rates used to generate the graph. Each supported protocol offers one or more options. Unused protocols can be omitted.": "Choose the traffic rates used to generate the graph. Each supported protocol offers one or more options. Unused protocols can be omitted.",
+  "Circuit Breaker": "Circuit Breaker",
+  "Click + Drag": "Click + Drag",
+  "Click kebab menu on a node label for links to details, traffic, inbound/outbound metrics and node graph.": "Click kebab menu on a node label for links to details, traffic, inbound/outbound metrics and node graph.",
+  "Click kebab menu on a node label to see the contextual menu options for the node.": "Click kebab menu on a node label to see the contextual menu options for the node.",
+  "Click on a node or edge to see its summary and emphasize its end-to-end paths.": "Click on a node or edge to see its summary and emphasize its end-to-end paths.",
+  "Close": "Close",
+  "Close Legend": "Close Legend",
+  "Close Replay": "Close Replay",
+  "Cluster": "Clúster",
+  "Compact view": "Vista compacta",
+  "Compare with similar traces": "Compare with similar traces",
+  "Concentric": "Concentric",
+  "Configuration": "Configuration",
+  "Configuration:": "Configuration:",
+  "Confirm": "Confirm",
+  "Contextual Menu": "Contextual Menu",
+  "Contextual menu on nodes": "Contextual menu on nodes",
+  "Control plane": "Control plane",
+  "Control plane metrics": "Control plane metrics",
+  "Copied": "Copied",
+  "Copy": "Copy",
+  "Copy all resources": "Copy all resources",
+  "cores": "cores",
+  "CPU": "CPU",
+  "Create": "Create",
+  "Current duration": "Current duration",
+  "Current refresh interval: {{refreshInterval}}": "Current refresh interval: {{refreshInterval}}",
+  "Current time range": "Current time range",
+  "custom": "custom",
+  "Custom": "Custom",
+  "Dagre": "Dagre",
+  "Data Plane": "Data Plane",
+  "Dataplane table": "Dataplane table",
+  "Debug information": "Debug information",
+  "Debug Information": "Debug Information",
+  "Debug information has been copied to your clipboard.": "Debug information has been copied to your clipboard.",
+  "Debug information was copied to your clipboard, but is outdated now. It could be caused by new data received by auto refresh timers.": "Debug information was copied to your clipboard, but is outdated now. It could be caused by new data received by auto refresh timers.",
+  "Degraded": "Degraded",
+  "Details in side panel on nodes": "Details in side panel on nodes",
+  "Direction": "Direction",
+  "Direction type is not valid.": "Direction type is not valid.",
+  "Disabled": "Disabled",
+  "Display": "Display",
+  "Display the legend to learn about what the different shapes, colors and backgrounds mean.": "Display the legend to learn about what the different shapes, colors and backgrounds mean.",
+  "Distributed Tracing": "Rastreo distribuido",
```

Comment:
I think it sounds better "Trazas distribuidas", but it is true I've also seen this translation in other articles.

Reply:
It sounds better, thanks @josunect :slightly_smiling_face:. Translating this stuff is not easy and is very subjective. I think we are very used to the English names haha.